modules/nf-neuro/reconst/qball/meta.yml

```
@@ -0,0 +1,162 @@
+    - mask:
+        type: file
+        description: Mask.
+        pattern: "*.{nii,nii.gz}"
+        mandatory: false
+        ontologies:
+          - edam: http://edamontology.org/format_4001 # NIFTI format
+output:
+  gfa:
+    - - meta:
+          type: map
+          description: |
+            Groovy Map containing sample information
+            e.g. `[ id:'test', single_end:false ]`
+      - "*__gfa.nii.gz":
+          type: file
+          description: Generalized fractional anisotropy.
+          pattern: "*__gfa.{nii,nii.gz}"
+          ontologies:
+            - edam: http://edamontology.org/format_4001 # NIFTI format
+  peaks:
+    - - meta:
+          type: map
+          description: |
+            Groovy Map containing sample information
+            e.g. `[ id:'test', single_end:false ]`
+      - "*__peaks.nii.gz":
+          type: file
+          description: Extracted peaks.
+          pattern: "*__peaks.{nii,nii.gz}"
+          ontologies:
+            - edam: http://edamontology.org/format_4001 # NIFTI format
+  peak_indices:
+    - - meta:
+          type: map
+          description: |
+            Groovy Map containing sample information
+            e.g. `[ id:'test', single_end:false ]`
+      - "*__peak_indices.nii.gz":
+          type: file
+          description: Generated peaks indices on the sphere.
+          pattern: "*__peak_indices.{nii,nii.gz}"
+          ontologies:
+            - edam: http://edamontology.org/format_4001 # NI
+  sh:
+    - - meta:
+          type: map
+          description: |
+            Groovy Map containing sample information
+            e.g. `[ id:'test', single_end:false ]`
+      - "*__sh.nii.gz":
+          type: file
+          description: Spherical harmonics coefficients.
+          pattern: "*__sh.{nii,nii.gz}"
+          ontologies:
+            - edam: http://edamontology.org/format_4001 # NIFTI format
+  nufo:
+    - - meta:
+          type: map
+          description: |
+            Groovy Map containing sample information
+            e.g. `[ id:'test', single_end:false ]`
+      - "*__nufo.nii.gz":
+          type: file
+          description: NUFO map.
+          pattern: "*__nufo.{nii,nii.gz}"
+          ontologies:
+            - edam: http://edamontology.org/format_4001 # NIFTI format
+  a_power:
+    - - meta:
+          type: map
+          description: |
+            Groovy Map containing sample information
+            e.g. `[ id:'test', single_end:false ]`
+      - "*__a_power.nii.gz":
+          type: file
+          description: Anisotropic power map.
+          pattern: "*__a_power.{nii,nii.gz}"
+          ontologies:
+            - edam: http://edamontology.org/format_4001 # NIFTI format
+  versions:
+    - versions.yml:
+        type: file
+        description: File containing software versions
+        pattern: versions.yml
+        ontologies:
+          - edam: http://edamontology.org/format_3750 # YAM
+authors:
+  - "@EliseCos and @arnaudbore"
```

Put that in two items please, else links in the docs won't work correctly.

```suggestion
  - @EliseCos
  - @arnaudbore
```